### DISCUSSION TITLE
cilium-handbook/concepts/security/identity/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/cloud-native-library#37] [Announcements]
# cilium-handbook/concepts/security/identity/

Kubernetes 等容器管理系统部署了一个网络模型，该模型为每个 pod（容器组）分配一个单独的 IP 地址。这确保了架构的简单性，避免了不必要的网络地址转换 （NAT），并为每个单独的容器提供了全范围的端口号以供使用。这种模

https://lib.jimmysong.io/cilium-handbook/concepts/security/identity/

**Comment by rootsongjc** [comment]
那些基于 IP 地址过滤的传统上的网络策略已经无法适应云原生环境，因为 Pod 的 IP 是动态的，必须要有一种新的方式来为应用赋予身份，比如用服务账户或者 SPIFFE。
